deep_qa/data/dataset_readers/squad_sentence_selection_reader.py

```diff
@@ -0,0 +1,99 @@
+import argparse
+import json
+import logging
+import os
+import random
+from tqdm import tqdm
+
+logger = logging.getLogger(__name__) # pylint: disable=invalid-name
+
+random.seed(2157)
+
+
+def main():
+    log_format = '%(asctime)s - %(name)s - %(levelname)s - %(message)s'
+    logging.basicConfig(level=logging.INFO, format=log_format)
+
+    parser = argparse.ArgumentParser(description=("Parse a SQuAD v1.1 data file for "
+                                                  "use by the SentenceSelectionInstance"))
+    parser.add_argument('input_filename', help='Input SQuAD json file.')
+    parser.add_argument('--output_directory',
+                        help=('Output directory. Make sure to end the string with a /'))
+    args = parser.parse_args()
+    reader = SquadSentenceSelectionReader(args.output_directory)
+    reader.read_file(args.input_filename)
+
+
+class SquadSentenceSelectionReader():
+    def __init__(self, output_directory=None):
+        self.output_directory = output_directory
+
+    def read_file(self, input_filepath):
+        # import is here because it is slow and not needed by default.
+        import nltk
+        processed_rows = []
+        logger.info("Reading file at %s", input_filepath)
+        with open(input_filepath) as dataset_file:
+            dataset_json = json.load(dataset_file)
+            dataset = dataset_json['data']
+        for article in tqdm(dataset):
+            for paragraph in article['paragraphs']:
+                context_article = paragraph["context"]
+                # split the context_article into a list of sentences
+                sentences = nltk.sent_tokenize(context_article)
```

Comment:
I think it does, but it involves downloading like 1GB of files, and I don't think speed is really an issue here